Source/Core/DiscIO/NANDImporter.cpp

```diff
@@ -241,7 +253,19 @@ bool NANDImporter::ExtractCertificates()
 
     const std::string pem_file_path = m_nand_root + std::string(certificate.filename);
     const ptrdiff_t certificate_offset = std::distance(content_bytes.begin(), search_result);
+    if (certificate_offset < 2 || static_cast<size_t>(certificate_offset) >= content_bytes.size())
+    {
+      ERROR_LOG_FMT(DISCIO, "ExtractCertificates: Invalid certificate offset {:#x}",
```

Comment:
Maybe add `content_bytes` size here as well?

Reply:
`ERROR_LOG_FMT(DISCIO, "ExtractCertificates: Certificate offset {:#x} must be between {} and {}", certificate_offset, min_offset, content_bytes);`